README.md

```diff
@@ -118,11 +122,13 @@ Publishes a SNAPSHOT jar whenever a feature branch is merged into the project's
 
 | Variable                	| Default Value                                                        	| Description                                                                                                                                                                               	|
 |-------------------------	|----------------------------------------------------------------------	|--------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------   |
-| STANDARD_GRADLE_FLAGS   	| -s --no-daemon -PnoMavenLocal --refresh-dependencies --console=plain 	| Default Gradle flags that will be appended to all Gradle commands                                                                                                                             |
+| STANDARD_GRADLE_FLAGS   	|           | -s --no-daemon -PnoMavenLocal --refresh-dependencies --console=plain  (-PsafeTest)	| Default Gradle flags that will be appended to all Gradle commands (Will include -PsafeTest when set)                                                                                        |
```

Comment:
We should probably call out the `SAFE_TEST` variable here if we're going to mention the flag.